HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoicesListViewModel.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  InvoicesListViewModel.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import UIKit
+import GiniHealthAPILibrary
+import GiniCaptureSDK
+import GiniBankAPILibrary
+
+struct DocumentWithExtractions: Codable {
+    var documentID: String
+    var amountToPay: String?
+    var paymentDueDate: String?
+    var recipient: String?
+    
+    init(documentID: String, extractionResult: GiniHealthAPILibrary.ExtractionResult) {
+        self.documentID = documentID
+        self.amountToPay = extractionResult.payment?.first?.first(where: {$0.name == "amount_to_pay"})?.value
+        self.paymentDueDate = extractionResult.extractions.first(where: {$0.name == "payment_due_date"})?.value
+        self.recipient = extractionResult.payment?.first?.first(where: {$0.name == "payment_recipient"})?.value
+    }
+    
+    init(documentID: String, extractions: [GiniBankAPILibrary.Extraction]) {
```

Comment:
[clarification]: I think we can use `GiniHealthAPILibrary.Extraction` instead `GiniBankAPILibrary.Extraction`. Is the same object. And then no need to import ` GiniBankAPILibrary`.

Reply:
Ok, I got it. Thanks for clarifications @razvancapra.